packages/styles/stories/BTP/Navigation/horizontal/navigation-horizontal-expander.example.html

```diff
@@ -1,19 +1,18 @@
 <div style="height: 500px">
     <nav class="fd-navigation fd-navigation--horizontal" aria-label="Horizontal Navigation">
         <div class="fd-navigation__container">
-            <ul class="fd-navigation__list" role="menubar" aria-label="Horizontal Navigation">
+            <ul class="fd-navigation__list" role="tablist" aria-orientation="horizontal" aria-label="Horizontal Navigation">
                 <li
                     role="none"
                     tabindex="-1"
                     class="fd-navigation__list-item fd-navigation__list-item--home"
-                    aria-hidden="true"
                 >
                     <div
                         class="fd-navigation__item"
                         aria-current="page"
                         aria-selected="false"
                         aria-expanded="false"
-                        role="menuitem"
+                        role="tab"
```

Comment:
Role `tab` needs the following:
- aria-labelledby: this should be leading to the span that wraps the text
- aria-controls: referring to the tabpanel that the tab controls
- aria-posinset and aria-setsize
- aria-selected
- aria-expanded
- aria-haspopup: if the tab has a popup menu

Reply:
https://www.w3.org/WAI/ARIA/apg/patterns/tabs/